internal/sdk/cloudian/sdk.go

```diff
@@ -143,18 +143,21 @@ func (client Client) ListUsers(ctx context.Context, groupId string, offsetUserId
 
 	limit := 100
 
-	offsetQueryParam := ""
-	if offsetUserId != nil {
-		offsetQueryParam = "&offset=" + *offsetUserId
-	}
-
-	url := client.baseURL + "/user/list?groupId=" + groupId + "&userType=all&userStatus=all&limit=" + strconv.Itoa(limit) + offsetQueryParam
-
-	req, err := client.newRequest(ctx, url, http.MethodGet, nil)
+	req, err := client.newRequest(ctx, "/user/list", http.MethodGet, nil)
```

Comment:
Maybe consider a library with a more fluent API? Example: https://github.com/earthboundkid/requests

I don't like the "spread" requests building. For this PR, we could supply the parameters as a map to `client.newRequest`?

Reply:
Might go there eventually, updated to use map for sending query params now